src/client.ts

```diff
@@ -236,11 +236,15 @@ export class Client {
    */
   async beat(): Promise<string> {
```

Comment:
This this method is a bit low-level, I would prefer for it to reject the promise when it fails sending the beat to the server. Callers of `Client#beat` could then catch that error where necessary. As it's written, a caller of `Client#beat` would have no way of knowing whether it succeeded or not, because it returns a truthy value either way.

Since `beat` is already wrapped in a try/catch in `Worker#work` as of 4.1.0, do you think it will be okay if we don't try/catch here?

Reply:
Thanks for your inputs, I will upgrade the package and test this try/catch in `Worker#work`

And I think stopping the worker during this error seems to be the best thing to do @jbielick 

I can update the PR later today

Reply:
Hey Josh, @jbielick 

`4.1.4` seems to handle the error signal in the catch block!